src/pages/orders/OrderList.tsx

```diff
@@ -1,69 +1,50 @@
 import React from 'react';
-import gql from 'graphql-tag';
 import { Pagination } from 'react-bootstrap';
-import OrderRow from './OrderRow';
-import { OrderList_orderPagination } from './__generated__/OrderList_orderPagination';
+import { OrderRow } from './OrderRow';
+import { OrderList_paginationFragment } from './__generated__/OrderList_pagination';
```

Comment:
**3) Will be nice `typescript-react-apollo` allows to turn off `Fragment` suffix** for generated types eg `OrderRow_orderFragment` -> `OrderRow_order`. [relay-compiler enforces fragments to be named as `<FileName>_<propName>`](https://relay.dev/docs/en/fragment-container#arguments). And it's is very useful and cool!